cpp/tests/CMakeLists.txt

```diff
@@ -576,7 +582,8 @@ if(BUILD_CUGRAPH_MG_TESTS)
     ###############################################################################################
     # - MG BETWEENNESS CENTRALITY tests -----------------------------------------------------------
     ConfigureTestMG(MG_BETWEENNESS_CENTRALITY_TEST centrality/mg_betweenness_centrality_test.cpp)
-    ConfigureTestMG(MG_EDGE_BETWEENNESS_CENTRALITY_TEST centrality/mg_edge_betweenness_centrality_test.cpp)
+    ConfigureTestMG(MG_EDGE_BETWEENNESS_CENTRALITY_TEST
```

Comment:
This seems like an odd edit.

Reply:
This exceeded the line length limit :-) Should we allow long lines in CMakeLists.txt?

Reply:
I didn't realize we had one.

Reply:
I guess we don't have one officially, but I implicitly consider the end of `###############################################################################################` as the line limit.